go-client/TestSetupAbstraction.go

```diff
@@ -0,0 +1,210 @@
+package exasol_test_setup_abstraction_go
+
+import (
+	"bytes"
+	"database/sql"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"github.com/exasol/exasol-driver-go"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+type TestSetupAbstraction struct {
+	server         *exec.Cmd
+	serverEndpoint string
+	stopped        *bool
+	stoppedMutex   *sync.Mutex
+}
+
+const serverVersion = "0.1.0"
+const serverJar = "exasol-test-setup-abstraction-server-" + serverVersion + ".jar"
+
+func Create(configFilePath string) TestSetupAbstraction {
```

Comment:
Does it make sense to provide a method without the config file path? It's optional anyway.

Reply:
basically yes. However, we don't have that method in the java part. So what we could do is create a version without config file that then calls the existing one with something like `nonExistingConfigFile.json`. That could make trouble if the file exsists. Thats unlikely but possible.